src/configs/linkedin-passport.ts

```diff
@@ -68,7 +68,7 @@ passport.deserializeUser(async (primary_email: string, done) => {
 const cookieExtractor = (req: Request): string => {
   let token = null
   if (req?.cookies) {
-    token = req.cookies.jwt
+    token = req.cookies.accessToken
```

Comment:
We need a refresh token endpoint as well. Refer to this
https://medium.com/@techsuneel99/jwt-authentication-in-nodejs-refresh-jwt-with-cookie-based-token-37348ff685bf

Reply:
added the separate refresh token endpoint

Reply:
added a seperate endpoint for a refresh token